portlist/portlist_macos.go

```diff
@@ -202,6 +209,7 @@ func (im *macOSImpl) addProcesses() error {
 			switch {
 			case m != nil:
 				m.port.Process = cmd
+				m.port.Pid = pid
```

Comment:
why is this only populated on mac?

Reply:
@bradfitz great catch actually, I initially rebased the pid part from https://github.com/tailscale/tailscale/pull/6202, but now I realize there are a few missing areas and not just linux/mac_os. For example [this line](https://github.com/tailscale/tailscale/blob/4e86857313f3acb370393aacbbbd4d47cba105c5/portlist/portlist.go#L72) should also make sure to copy over the pid